### ISSUE TITLE
TypeScript error TS7016: Could not find a declaration file for module 'react-native-marked'

### ISSUE THREAD

**Issue by derRaab** [gmsgowtham/react-native-marked#154]
First of all - thanks for all your great work! 🌻

I ran into an TypeScript compiler error and even this might be a stupid TypeScript setup issue on our side (I'm sadly no TypeScript wizzard), I hope this basic patch to the node module will help others as well.

So this was the error:

```
error TS7016: Could not find a declaration file for module 'react-native-marked'. '/...project/node_modules/react-native-marked/dist/commonjs/index.js' implicitly has an 'any' type.
  Try `npm i --save-dev @types/react-native-marked` if it exists or add a new declaration (.d.ts) file containing `declare module 'react-native-marked';
```
I "fixed" it by adding a ìndex.d.ts` file to the root of your package containing these lines:
```
import Markdown from "./dist/typescript/src/lib/Markdown";
export default Markdown;
```
And then I did patch the node module using `npx patch-package react-native-marked` so it will aways be applied.

I wonder if anyone ever runs into this, if it would make sense to add this file to the repository or if this is just a stupid setup error on our side? Thanks! :)

**Comment by gmsgowtham** [comment]
Thanks for reporting the issue @derRaab. I was able to re-produce the issue and published a fix under [v1.6.11](https://github.com/gmsgowtham/react-native-marked/releases/tag/v1.6.11). Please re-open the issue if not resolved at your end.

**Comment by derRaab** [comment]
@gmsgowtham Thank you so much - works like a charm. And I even learned that the package.json does reference the type definitions. :)
